### PR TITLE
TRD-558 Fallback for Fetching Trace Call

### PR DESCRIPTION
**Pull request by linhnt3400**

### CODE REVIEW

File: pkg/rpcnode/client.go [hunk 1]
```diff
@@ -8,12 +8,14 @@ import (
 )
 
 type Client struct {
-	ethClient *ethclient.Client
+	ethClient         *ethclient.Client
+	fallbackETHClient *ethclient.Client
 }
 
-func NewClient(ethClient *ethclient.Client) *Client {
+func NewClient(ethClient, fallbackClient *ethclient.Client) *Client {
```
Comment (ngocthanh1389) [inline pkg/rpcnode/client.go:15]: using variadic for eth client, so user can pass arbitrary number of eth client
```suggestion
func NewClient(ethClient... *ethclient.Client) *Client {
```

File: internal/app/rpcnode.go [hunk 2]
```diff
@@ -13,10 +14,17 @@ var (
 		Usage:  "RPC node url",
 		Value:  "https://ethereum.kyberengineering.io/trading-tokyo",
 	}
+	FallbackRPCUrlFlag = &cli.StringFlag{
```
Comment (ngocthanh1389) [inline internal/app/rpcnode.go:17]: think should modify old string flag to  string slice flag

File: pkg/rpcnode/client.go (hunk 1 above)
Comment (linhnt3400) [reply pkg/rpcnode/client.go]: updated

File: internal/app/rpcnode.go (hunk 2 above)
Comment (linhnt3400) [reply internal/app/rpcnode.go]: updated

**Review by ngocthanh1389 — APPROVED** [review]
LGTM

**Pull request merged**